src/mmw/js/src/modeling/tr55/quality/templates/table.html

```diff
@@ -2,7 +2,7 @@
     <thead>
         <tr>
             <th>Quality Measure</th>
-            <th class="text-left">Load (lbs)</th>
+            <th class="text-left">Load (Kg)</th>
```

Comment:
SI representation uses all lowercase letters, so "kg" instead of "Kg". See https://en.wikipedia.org/wiki/Kilogram